docs/samples/calling/app.js

```diff
@@ -209,10 +207,58 @@ function initWebex(e) {
     };
   }
 
-  webex = window.webex = Webex.init(webexConfig);
+  const clientConfig = {
+    calling: true,
+    contact: true,
+    callHistory: true,
+    callSettings: true,
+    voicemail: true,
+  }
+
+  const loggerConfig = {
+    level: 'info'
+  }
+
+  const {region, country} = registrationForm.elements;
+
+  const serviceData = {indicator: 'calling', domain: ''};
+
+  if (serviceIndicator.value) {
+    serviceData.indicator = serviceIndicator.value;
+  }
+
+  if (serviceDomain.value) {
+    serviceData.domain = serviceDomain.value;
+  }
+
+  const callingClientConfig = {
+    logger: loggerConfig,
+    discovery: {
+      region: region.value,
+      country: country.value,
+    },
+    serviceData,
+  };
+
+  if (callingClientConfig.discovery.country === 'Country') {
+    callingClientConfig.discovery.country = '';
+  }
+
+  if (callingClientConfig.discovery.region === 'Region') {
+    callingClientConfig.discovery.region = '';
+  }
+
+  const callingConfig = {
+    clientConfig : clientConfig,
+    callingClientConfig: callingClientConfig,
+    logger:loggerConfig
+  }
+
+  calling = new Calling({options: webexConfig, callingConfig: callingConfig});
```

Comment:
rather then naming it options lets name it as webexConfig

Reply:
done